pkg/config/config.go

```diff
@@ -112,16 +114,27 @@ type SecurityCapabilityLists struct {
 
 // ParseFile parses config from a file.
 func ParseFile(path string) (Configuration, error) {
-	configBox := packr.New("Config", "../../examples")
-	var rawBytes []byte
 	var err error
-	if path == "" {
-		rawBytes, err = configBox.Find("config.yaml")
+	var rawBytes []byte
+
+	if strings.HasPrefix(path, "https://") || strings.HasPrefix(path, "http://") {
+		//path is a url
+		response, err := http.Get(path)
+		if err != nil {
+			return Configuration{}, err
+		}
+		rawBytes, err = ioutil.ReadAll(response.Body)
 	} else {
-		rawBytes, err = ioutil.ReadFile(path)
-	}
-	if err != nil {
-		return Configuration{}, err
+		//path is local
+		configBox := packr.New("Config", "../../examples")
+		if path == "" {
+			rawBytes, err = configBox.Find("config.yaml")
+		} else {
+			rawBytes, err = ioutil.ReadFile(path)
+		}
+		if err != nil {
```

Comment:
This needs to go outside the bigger `if` block, to handle the case where `ioutil.ReadAll` errors out